pool/src/constants.rs

```diff
@@ -8,3 +8,6 @@ pub const SCALAR_7: i128 = 1_0000000;
 
 // seconds per year
 pub const SECONDS_PER_YEAR: i128 = 31536000;
+
+// approximate week in blocks assuming 5 seconds per block
+pub const WEEK_IN_BLOCKS: u32 = 120960;
```

Comment:
A week seems like a really long time, no?

Consider a DAO doing this:

1. Create proposal for queueing the `init_reserve` (2 days before voting, 3 day voting period, 2 day timelock) (7 days)
2. Wait ~ 1 day
3. Create another proposal for submitting the init (7 day gov period)

So its a 15 day, 2 proposal process to add a reserve to the pool, assuming they are smart enough to double dip the timelock and proposal period.

Reply:
Will most DAO's have a mandatory 2 day period before voting? I made initialization permissionless so I think it's important the queue time is longer than a typical DAO vote -> execution time so they can cancel initializations if a malicious one snuck in without anyone noticing